<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallWidget</class>
 <widget class="QWidget" name="CallWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="callFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridCallInfo">
        <item row="0" column="1">
         <widget class="QLabel" name="lbName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Display Name</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="lbPhoto">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Photo</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Phone Number</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QFrame" name="frZrtp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lbZrtpInfo">
             <property name="text">
              <string>Sec</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lbZrtpCompare">
             <property name="text">
              <string>Compare:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="lbZrtpSAS">
             <property name="text">
              <string>SAS</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbZrtpVerify">
             <property name="text">
              <string>Verif</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4" rowspan="2">
            <widget class="QPushButton" name="btZrtpConfirm">
             <property name="text">
              <string>Confirm</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5" rowspan="2">
            <widget class="QPushButton" name="btZrtpReset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="lbDTMF">
          <property name="text">
           <string>DTMF</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lbStatus">
            <property name="text">
             <string>Calling</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbTime">
            <property name="text">
             <string>0:00:00</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frRegularCall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QPushButton" name="btHold">
           <property name="text">
            <string>Hold</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btDialpad">
           <property name="text">
            <string>Dialpad</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btTransfer">
           <property name="text">
            <string>Transfer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btRecord">
           <property name="text">
            <string>Record</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btMute">
           <property name="text">
            <string>Mute</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QPushButton" name="btHangup">
           <property name="text">
            <string>End Call</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btSpeaker">
           <property name="text">
            <string>Speaker</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frConfCall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btMuteC">
           <property name="text">
            <string>Mute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btKick">
           <property name="text">
            <string>Kick</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btDialpadC">
           <property name="text">
            <string>Dialpad</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btHangupC">
           <property name="text">
            <string>End Call</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frInboundCall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btAnswer">
           <property name="text">
            <string>Answer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSilence">
           <property name="text">
            <string>Ignore</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btReject">
           <property name="text">
            <string>Reject</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFailedCall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btCallBack">
           <property name="text">
            <string>Call Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frOutboundCall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btHangupO">
           <property name="text">
            <string>End Call</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btAnswer</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btMute</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSilence</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btHold</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btHangup</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btReject</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btKick</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btRecord</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallBack</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSpeaker</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDialpad</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btTransfer</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btMuteC</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDialpadC</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btHangupC</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btHangupO</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btZrtpConfirm</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btZrtpReset</sender>
   <signal>clicked()</signal>
   <receiver>CallWidget</receiver>
   <slot>buttonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClicked()</slot>
 </slots>
</ui>
